cirq/study/sweepable.py

```diff
@@ -14,16 +14,18 @@
 
 """Defines which types are Sweepable."""
 
-from typing import Dict, Iterable, Iterator, List, Union, cast
+from typing import Dict, Iterable, Iterator, List, Union, cast, Any
 import itertools
 
 from cirq._doc import document
 from cirq.study.resolver import ParamResolver, ParamResolverOrSimilarType
 from cirq.study.sweeps import ListSweep, Points, Sweep, UnitSweep, Zip
 
 
-Sweepable = Union[Dict[str, float], ParamResolver, Sweep, Iterable[
-    Union[Dict[str, float], ParamResolver, Sweep]], None]
+Sweepable = Union[Dict[str, Union[float, str, Iterable, Any]], ParamResolver,
```

Comment:
Oh, yikes.  This is a bit tangential to the review, but I think this deserves a line comment since this type has become very difficult to read.

Reply:
Done.